translator/config/schema_test.go

```diff
@@ -12,9 +12,9 @@ import (
 )
 
 func TestGetJsonSchema(t *testing.T) {
-	jsonFile, err := ioutil.ReadFile("./schema.json")
-	if err != nil {
-		panic(err)
+	jsonFile, e := ioutil.ReadFile("./schema.json")
+	if e != nil {
+		panic(e)
```

Comment:
This shouldn't panic, but I'm going to clean up our test code so you don't have to change it.
```suggestion
	assert.NoError(t, e)
```

Reply:
Agree. Thought of doing in another PR but prefer left it as it is.